project_forecast_line/models/project_project.py

```diff
@@ -11,8 +13,10 @@ def _update_forecast_lines_trigger_fields(self):
 
     def write(self, values):
         res = super().write(values)
-        written_fields = list(values.keys())
+        written_computed_fields = get_written_computed_fields(
+            self, tuple(sorted(values))
+        )
```

Comment:
Stored computed fields updates don't go through `write`, but they do go through `_write`.
You could override that method instead, and thus you avoid maintaing this complex utility method

Reply:
thanks, @ivantodorovich for the suggestion. For the moment I've added it in TODO, I'll check when have time.